test/functional/tests/cli/test_cli_help_spelling.py

```diff
@@ -0,0 +1,230 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import pytest
+import time
+
+from api.cas import casadm
+from api.cas.cli import casadm_bin
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_utils.size import Unit, Size
+
+
+def test_cli_help_spelling():
+    """
+    title: Spelling test for 'help' command
+    description: Validates spelling of 'help' in CLI
+    pass criteria:
+    - no spelling mistakes are found
+    """
+
+    personal_directory = "/root/open-cas-linux/test/smoke_test/cas_ex.en.pws"
+
+    with TestRun.step("Run aspell"):
+        output = TestRun.executor.run_expect_success(
+            f"{casadm_bin} -H 2>&1 | aspell list --lang=en_US "
+            f"--add-extra-dicts={personal_directory}")
+
+        if output.stdout:
+            TestRun.LOGGER.error("Misspelled words found:\n")
+            TestRun.LOGGER.error(output.stdout)
+
+        first = "{ cmd=1;next }"
+        second = "{ cmd=0 }"
+        third = "{ print $0 }"
+        backslash = "\\"
+
+        commands = TestRun.executor.run_expect_success(
+            f"{casadm_bin} -H | awk '/Available commands:/{first} /For detailed help/ {second} "
+            f"cmd {third}' | grep -o '{backslash}-{backslash}-{backslash}S*'")
+
+        commands = str(commands).split()
+        commands = commands[3:-1]
+
+        output = TestRun.executor.run_expect_success(
+            f"{casadm_bin} {commands} -H | aspell list --lang=en_US "
+            f"--add-extra-dicts={personal_directory}")
+        if output.stdout:
+            TestRun.LOGGER.error("Misspelled words found:\n")
+            TestRun.LOGGER.error(output.stdout)
+
+    return 0
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.nand, DiskType.optane]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+@pytest.mark.CI()
+def test_cli_write_mode():
+    """
+    title: Verification test for write mode: read only
+    description: Verify if write mode: read only, works as expected and cache only reads
+    and do not cache writes
+    pass criteria:
+    - cache does not cache write
+    """
```

Comment:
This test doesn't check spelling. It should probably be put somewhere in "ci" or "cache_ops" folder.